aleph-client/src/treasury.rs

```diff
@@ -0,0 +1,152 @@
+use crate::{try_send_xt, wait_for_event, AnyConnection, RootConnection, SignedConnection};
```

Comment:
In general to this file: what do you think to make `pub fn`s organized as first in this file, followed by private `fn`? Motivation is for a reader which usually wants to read public API of this module first

Reply:
This is also my preferred order (public functions first, and then the functions used by them, toposorted), but I was under the impression that most files in our codebase toposort the other way and was keeping to that. Is there any agreed practice in place?

Reply:
I reordered it; this is also my taste, but some time ago both reviewers were opting for absolute toposort, hence this previous outcome; but we should fight for our feelings!

Reply:
Honestly, I feel like `rustfmt` or, at the most, `clippy` should just enforce the chosen toposort :)

Reply:
Yes, we should at some point enforce this by the tooling.